src/zenml/integrations/aws/container_registries/aws_container_registry.py

```diff
@@ -76,10 +113,9 @@ def prepare_image_push(self, image_name: str) -> None:
             raise ValueError(f"Invalid docker image name '{image_name}'.")
         repo_name = match.group(1)
 
+        client = self._get_ecr_client()
```

Comment:
client might be None

Reply:
Currently it can not be None, see explanation above